lambda_layers/core_layer/python/core_layer/helper.py

```diff
@@ -39,7 +39,7 @@ def get_date_time_str(dt):
         return dt
 
 
-def set_cors(response, event):
+def set_cors(response, event, plugin_call=False):
```

Comment:
Finde den Namen des Parameter etwas zu spezifisch. Der Parameter sollte außerdem in die Code Doku aufgenommen werden.